openlibrary/templates/lists/widget.html

```diff
@@ -93,6 +93,15 @@
 
 $jsdef render_widget_add(lists, work_key, edition_key, users_work_read_status, username, pageurl, readinglogonly):
     <div class="dropit">
+        $if pidentifier:
```

Comment:
CSS embedded in JS seems like an anti-pattern, but will defer to @jdlrobson for best practices.

Reply:
Yeh this should be avoided at all costs.

This is also a separation of concerns since the list it's hiding is elsewhere - present in openlibrary/templates/publishers/view.html and openlibrary/templates/subjects.html - those templates should be making this decision. The widget macro/template shouldn't have to be concerned about that.

Instead of displaying none, let's not output that element in those files. It should simplify a lot of this code.